src/aiida_quantumespresso/workflows/pdos.py

```diff
@@ -451,11 +453,15 @@ def run_nscf(self):
 
         if 'scf' in self.inputs:
             inputs.pw.parent_folder = self.ctx.scf_parent_folder
+        else:
```

Comment:
Do we need this `else` here? If we use `inputs.pw.parent_folder` to obtain the output parameters via the creator on line 464, we don't need the `self.ctx.scf_parent_folder`, I believe?